src/GraphQL/Types/Composite/ComplexGraphType.cs

```diff
@@ -342,7 +342,7 @@ public virtual FieldType AddField(FieldType fieldType)
             string name,
             string? description = null,
             QueryArguments? arguments = null,
-            Func<IResolveFieldContext<TSourceType>, object?>? resolve = null,
+            Func<IResolveFieldContext<TSourceType>, object?>? resolve = null, // TODO: remove?
```

Comment:
I doubt this parameter makes sense. I see no usages of these two methods and for top-level node of subscription it should always return source object (I think). I propose to remove `resolve` parameter or even remove both methods since we have `ResolveStream/ResolveStreamAsync` extensions and these two methods IMO only clutter public API.

Reply:
Remove in develop of course after merging this one into master.